src/lib/features/project/project-store.ts

```diff
@@ -579,32 +581,84 @@ class ProjectStore implements IProjectStore {
     }
 
     async getApplicationsByProject(
-        projectId: string,
-    ): Promise<IProjectApplication[]> {
+        params: IProjectApplicationsSearchParams,
+    ): Promise<IProjectApplications> {
+        const { project, limit, sortOrder, sortBy, searchParams, offset } =
+            params;
+        const validatedSortOrder =
+            sortOrder === 'asc' || sortOrder === 'desc' ? sortOrder : 'asc';
         const query = this.db
             .with('applications', (qb) => {
                 qb.select('project', 'app_name', 'environment')
                     .distinct()
                     .from('client_metrics_env as cme')
                     .leftJoin('features as f', 'cme.feature_name', 'f.name')
-                    .where('project', projectId);
+                    .where('project', project);
```

Comment:
We should decide on a standard here throughout the codebase, I think it's potentially confusing that we sometimes use `project` and sometimes `projectId`.

Reply:
The feature search uses project in its filters, so followed that.